src/plays/react-multistep-form/react-multistep-form.css

```diff
@@ -0,0 +1,62 @@
+.btn {
```

Comment:
Do not override the selectors directly. Add an unique class name at the play details level and use that to prefix all the styeles.